src/poke_env/ps_client/server_configuration.py

```diff
@@ -1,10 +1,13 @@
 """This module contains objects related to server configuration.
 """
-from collections import namedtuple
+from typing import NamedTuple
+
+
+class ServerConfiguration(NamedTuple):
+    server_url: str
+    authentication_url: str
+
 
-ServerConfiguration = namedtuple(
-    "ServerConfiguration", ["server_url", "authentication_url"]
-)
 """Server configuration object. Represented with a tuple with two entries: server url
```

Comment:
This should be move as a class docstring

Reply:
0be3b9fdb0bfac29ae69c2db1307b8f394477600